### ISSUE TITLE
make clearer that multi-allele also refers to mutation in more than one gene

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#969]
Feedback from Per:

- Regarding double mutants: now I see how you intended them to be
annotated, and then it is logical to enter them as "multi-allele". For me,
"multi-allele" made me think of multiple alleles of the same gene, however.
How about naming this feature instead "mutations in more than one gene" or
similar, to make this clear?

I don't think we had any great suggestions in the long thread.....but can chat about on Thurs

**Comment by ValWood** [comment]
This was my rubbish suggestion, which was vetoed.
I don't think we had any more suggestions.
Maybe we just need some help/ mouseover text?

I still prefer "multi-gene alleles", I think that more people will understand  that this is where to put alleles of multiple gene, and we can cope with the rarer cases (Especially if the help tab explained this) where we need to represent "multiple alleles of the same gene"

How about something like

Phenotype (specify genotype details, then add a phenotype)

    Single-gene allele (single or concurrent changes  in one gene)
    Multiple-gene allele (changes to multiple genes or different copies of one gene)


instead of
  Phenotype

    Single allele (add allele details and phenotype)
    Multiple allele (specify a genotype, and then add a phenotype)

**Comment by ValWood** [comment]
Also want to prevent
![prevent](https://cloud.githubusercontent.com/assets/7359272/9878029/6fac64ac-5bb8-11e5-91c5-51788bc159ab.jpg)

**Comment by ValWood** [comment]
Also, I think it is going to be hard to stop people doing
multigene for   tmp1-A123E, tmp1-K124G
for example

We need to think about ways of  stopping people doing multigene as single gene and single gene as multigene....

**Comment by ValWood** [comment]
If you arrive at this page from "add a genotype"

The blue bar says
Specify the alleles of the new  genotype 

If you arrive at this page from "Edit"
Add, edit or delete alleles until the genotype is correctly specified

**Comment by mah11** [comment]
Note: we have not talked about the "Single allele (add allele details and phenotype)" and "Multiple allele (specify a genotype, and then add a phenotype)" wordings. I think they shouldn't change. Re:

> I still prefer "multi-gene alleles" ...

In an email, I touched briefly on the problem with the phrase "multi-gene alleles": at first glance, it simply doesn't make sense, and upon further consideration it gives the impression that each allele has to have parts from more than one gene. That's only the case for fusions.

In other words, "allele" does not mean the same thing as "genotype", and it's the "genotype" meaning that we need here.

**Comment by ValWood** [comment]
We decided that this is a documentation issue

**Comment by ValWood** [comment]
New ticket
https://github.com/pombase/canto/issues/1077
